SampleComponentLibraryApp/SampleComponentLibraryApp/ContextualHintView/ContextualHintViewViewController.swift

```diff
@@ -69,11 +73,23 @@ class ContextualHintViewViewController: UIViewController, Themeable {
 
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
-        let targetSize = CGSize(width: 350, height: UIView.layoutFittingCompressedSize.height)
-        preferredContentSize = hintView.systemLayoutSizeFitting(targetSize)
+        let targetSize = CGSize(width: UX.contextualHintWidth, height: UIView.layoutFittingCompressedSize.height)
+        var systemSize = hintView.systemLayoutSizeFitting(targetSize)
+        if UIDevice.current.orientation.isLandscape {
+            systemSize.width += UX.contextualHintLandscapeExtraWidth
+        }
+        preferredContentSize = systemSize
     }
 
     // MARK: Themeable
 
     func applyTheme() {}
+
+    // MARK: View Transitions
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        coordinator.animate(alongsideTransition: nil) { _ in
```

Comment:
Just noticed, should maybe `weakify` here?
```suggestion
        coordinator.animate(alongsideTransition: nil) { [weak self] _ in
```